front-end/src/components/modal/create/GraphSettings.js

```diff
@@ -1,14 +1,21 @@
 import '../../../styles/GraphSettingsStyles.css';
 
-const GraphSettings = ({ movePage, setGraphType, setLiveCheck }) => {
-
-  const graphType = () => {
-    setGraphType(document.getElementById("graphTypeSelect").value);
-
+const GraphSettings = ({ movePage, setGraphType, setLiveCheck, selectedVideo }) => {
+
+  const handleTypeSelect = (e) => {
```

Comment:
I feel like handleTypeSelect is not the right name for this function, since all your doing is adjusting the livedatacheckbox and not the graph type.

Reply:
The name gives context to the event that is passed in, as this is called each time the graph type select is updated. Does that make sense, or do you still think it should be changed?

Reply:
hmmm, I guess that makes sense. Ngl I think i misunderstood how this function works. I think this is ok now.